### DISCUSSION TITLE
Add the Web Share Target API to the PWA

### DISCUSSION THREAD

**Discussion by hkamran80** [WebCrateApp/webcrate#44] [Ideas] [answered]
Would it be possible to add the [Web Share Target API](https://web.dev/web-share-target/) to the WebCrate PWA? It would make it easier to add links to our crates.

I can work on this feature, if you want.

**Comment by BetaHuhn** [comment, marked as answer]
Yes, this was already requested a while ago, see https://github.com/WebCrateApp/webcrate/discussions/29#discussioncomment-1381791. I think this would be useful as well.

Feel free to work on this! Take a look at the [CONTRIBUTING.md](https://github.com/WebCrateApp/webcrate/blob/master/CONTRIBUTING.md) guide on how to work with the WebCrate codebase.

**Reply by hkamran80** [reply]
I'm trouble with a Deta Micro I setup to test WebCrate. It isn't loading any of the images. I can view them locally, but as soon as I deploy them to Deta, I get the error: `The image “https://[subdomain].deta.dev/banner.png” cannot be displayed because it contains errors.` Do you know why this is happening?

**Reply by BetaHuhn** [reply]
To serve images from express on Deta, you need to set an environment variable:

`BINARY_CONTENT_TYPES=image/*`

Refer to the Deta Docs on how to set the environment variable.

I should have included this in the CONTRIBUTING.md guide, sorry!

**Reply by hkamran80** [reply]
Gotcha, thank you!

**Reply by hkamran80** [reply]
Created the pull request: #46

**Reply by BetaHuhn** [reply]
This is included in version [1.4.1](https://github.com/WebCrateApp/webcrate/releases/tag/v1.4.1) :tada:
